base_user_gravatar/models/res_users.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# © 2015 Endika Iglesias
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api
+from openerp.exceptions import Warning
+from openerp.tools.translate import _
+import base64
+import hashlib
+import urllib2
+
```

Comment:
Executing `isort res_users.py` the order import is:
```python
 import base64
 import hashlib
 import urllib2

from openerp import api, models
from openerp.exceptions import Warning
from openerp.tools.translate import _
```
More infor [here](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports)